roles/vector/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
+vector_template: vector.toml.j2
```

Comment:
Setting this variable won't do anything unless you make some changes here https://github.com/dzervas/ansible-vector/blob/master/roles/vector/tasks/main.yml#L38

Reply:
Very true.. Not sure how I missed that.